CMakeLists.txt

```diff
@@ -203,6 +203,34 @@ if(ENABLE_LIBPNG_SUPPORT)
     list(APPEND LIBRARY_SOURCES Auxiliary/DirectXTexPNG.cpp)
 endif()
 
+macro(call_mingw_fxc work_dir input_filename output_filename entry_point var_name semantic _targetname)
+    add_custom_command(
+        COMMAND ${CMAKE_COMMAND} -DWORK_DIR=${work_dir} 
```

Comment:
General note: Please make sure you convert all TABs to SPACES and TRIM WHITESPACE.